package.json

```diff
@@ -60,7 +60,8 @@
     ],
     "coverageReporters": [
       "text",
-      "cobertura"
+      "cobertura",
+      "html"
```

Comment:
is this needed?

Reply:
For my local sanity. Not really for production